src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -350,6 +352,47 @@ void *frontendservicesesion(void *dummyPt) {
                 frontend_logger.log("Graph does not exist or cannot be deleted with the current hosts setting",
                                     "error");
             }
+
+        } else if (line.compare(REFORMAT) == 0) {
+            write(connFd, SEND.c_str(), FRONTEND_COMMAND_LENGTH);
+            write(connFd, "\r\n", 2);
+
+            // We get the name and the path to graph as a pair separated by |.
+            char graph_data[300];
+            bzero(graph_data, 301);
+            string name = "";
+            string path = "";
+
+            read(connFd, graph_data, 300);
+
+            string gData(graph_data);
+
+            Utils utils;
+            gData = utils.trim_copy(gData, " \f\n\r\t\v");
+            frontend_logger.log("Data received: " + gData, "info");
+
+            std::vector<std::string> strArr = Utils::split(gData, '|');
+
+            if (strArr.size() != 2) {
+                frontend_logger.log("Message format not recognized", "error");
+                break;
+            }
+
+            name = strArr[0];
+            path = strArr[1];
+
+            if (utils.fileExists(path)) {
+                std::cout << "Path exists" << endl;
+
+                JSONParser *jsonParser = new JSONParser();
+                jsonParser->readFile(path);
+                frontend_logger.log("Reformatted files created on /home/.jasminegraph/tmp/JSONParser/output", "info");
```

Comment:
Do you clean up the temporary files created in the intermediate folders?
For example, do you clean the temporary files created in "/home/.jasminegraph/tmp/JSONParser/output". If you do not, please add code to clean up those files so that we do not waste disk space unnecessarily.

Reply:
No I don't do it. I'll add the code for that

Reply:
I've done this